### PR TITLE
WF-65756-Loading and Saving of Word document in DocIO in UWP, Xamarin, WinUI, .NET MAUI, Linux and Mac platforms

### PR DESCRIPTION
**Pull request by ghost**
<table>
<tr>
<td>
Task Link
</td>
<td>
<a href="https://syncfusion.atlassian.net/browse/WF-65756">https://syncfusion.atlassian.net/browse/WF-65756</a>
</td>
</tr>
<tr>
<td>
Description
</td>
<td>
Add open and save codes of UWP platform
</td>
</tr>
<td>
Content review confirmation
</td>
<td>
Done
</td>
</tr>
</table>

### CODE REVIEW

File: File-Formats-toc.html [hunk 1]
```diff
@@ -123,7 +123,13 @@
 				<ul>
 					<li><a href="/file-formats/DocIO/Open-and-save-Word-document-in-ASP-NET-MVC">ASP.NET MVC</a></li>
 					<li><a href="/file-formats/DocIO/Open-and-save-Word-document-in-ASP-NET">ASP.NET</a></li>
+					<li><a href="/file-formats/DocIO/Open-and-save-Word-document-in-Xamarin">Xamarin</a></li>
```
Comment (MohanaselvamJothi) [inline File-Formats-toc.html:126]: Rebase the branch

File: File-Formats/DocIO/Open-and-save-Word-document-in-WinUI.md [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+//Save a Word document to the stream.
+using MemoryStream outputStream = new();
+document.Save(outputStream, FormatType.Docx);
+//Save the stream as a Word document file in the local machine.
+Save(outputStream, "Sample.docx");
+{% endhighlight %}
+
+{% endtabs %}
+
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Read-and-Save-document/Open-and-save-Word-document/WinUI/WinUI-Desktop-app).
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Open-and-save-Word-document-in-WinUI.md:111]: Does this sample available? If not please remove this line

File: File-Formats/DocIO/Open-and-save-Word-document-in-WinUI.md [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+{% endhighlight %}
+
+{% endtabs %}
+
+Step 7: Add a new action method **OpenAndSaveDocument** in MainWindow.xaml.cs and include the below code snippet to **open an existing Word document in WinUI Desktop app**.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+private async void OnButtonClicked(object sender, RoutedEventArgs e)
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Open-and-save-Word-document-in-WinUI.md:67]: In sentence, method name mentioned as OpenAndSaveDocument, which is totally different in code example. Please modify the code example as 

  
Assembly assembly = typeof(Open_and_save_Word_document).GetTypeInfo().Assembly;
using (WordDocument document = new WordDocument(assembly.GetManifestResourceStream("Open_and_save_Word_document.Assets.Input.docx"), FormatType.Docx))

File: File-Formats/DocIO/Open-and-save-Word-document-in-WinUI.md [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+Step 4: Install the [Syncfusion.DocIO.NET](https://www.nuget.org/packages/Syncfusion.DocIO.Net) NuGet package as a reference to your project from the [NuGet.org](https://www.nuget.org/).
+
+N> Starting with v16.2.0.x, if you reference Syncfusion assemblies from trial setup or from the NuGet feed, you also have to add "Syncfusion.Licensing" assembly reference and include a license key in your projects. Please refer to this [link](https://help.syncfusion.com/common/essential-studio/licensing/overview) to know about registering a Syncfusion license key in your application to use our components.
+
+Step 5: Add a new button to the **MainWindow.xaml** as shown below.
+
+{% tabs %}
+
+{% highlight XML %}
+<Window
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Open-and-save-Word-document-in-WinUI.md:34]: <Window
    x:Class="OpenAndSave.MainWindow"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:local="using:OpenAndSave"
    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
    mc:Ignorable="d">

    <StackPanel Orientation="Horizontal" HorizontalAlignment="Center" VerticalAlignment="Center">
        <Button x:Name="button" Click="OpenAndSaveDocument">Open and save Word document</Button>
    </StackPanel>
</Window>

File: File-Formats/DocIO/Open-and-save-Word-document-in-WinUI.md [hunk 5]
```diff
@@ -0,0 +1,267 @@
...
+
+N> Starting with v16.2.0.x, if you reference Syncfusion assemblies from trial setup or from the NuGet feed, you also have to add "Syncfusion.Licensing" assembly reference and include a license key in your projects. Please refer to this [link](https://help.syncfusion.com/common/essential-studio/licensing/overview) to know about registering a Syncfusion license key in your application to use our components.
+
+Step 5: Add a new button in the **MainPage.xaml** as shown below.
+
+{% tabs %}
+
+{% highlight XML %}
+
+<Page
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Open-and-save-Word-document-in-WinUI.md:137]: <Page
    x:Class="OpenAndSave.MainPage"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:local="using:OpenAndSave1"
    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
    mc:Ignorable="d"
    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">

    <StackPanel Orientation="Horizontal" HorizontalAlignment="Center" VerticalAlignment="Center">
        <Button x:Name="button" Click="OpenAndSaveDocument">Open and save Word document</Button>
    </StackPanel>
</Page>

File: File-Formats/DocIO/Open-and-save-Word-document-in-MAUI.md [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+Step 3: Install the [Syncfusion.DocIO.NET](https://www.nuget.org/packages/Syncfusion.DocIO.Net) NuGet package as a reference to your project from [NuGet.org](https://www.nuget.org/).
+
+N> Starting with v16.2.0.x, if you reference Syncfusion assemblies from trial setup or from the NuGet feed, you also have to add "Syncfusion.Licensing" assembly reference and include a license key in your projects. Please refer to this [link](https://help.syncfusion.com/common/essential-studio/licensing/overview) to know about registering a Syncfusion license key in your application to use our components.
+
+Step 4: Add a new button to the **MainPage.xaml** as shown below.
+
+{% tabs %}
+
+{% highlight XML %}
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Open-and-save-Word-document-in-MAUI.md:31]: <ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
            xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
            x:Class="OpenAndSave.MainPage"
            BackgroundColor="{DynamicResource SecondaryColor}">
    <ScrollView>
        <Grid RowSpacing="25" RowDefinitions="Auto,Auto,Auto,Auto,*"
            Padding="{OnPlatform iOS='30,60,30,30', Default='30'}">
            <Button 
                Text="Open and save"
                FontAttributes="Bold"
                Grid.Row="0"
                SemanticProperties.Hint="Creates Word document you click"
                Clicked="OpenAndSaveDocument"
                HorizontalOptions="Center" />
        </Grid>
    </ScrollView>
</ContentPage>

File: File-Formats/DocIO/Open-and-save-Word-document-in-Mac.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+document.Save(outputStream, FormatType.Docx);
+//Close the Word file.
+document.Close();
+outputStream.Flush();
+outputStream.Dispose();
+{% endhighlight %}
+
+{% endtabs %}
+
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Read-and-Save-document/Open-and-save-Word-document/Mac).
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Open-and-save-Word-document-in-Mac.md:79]: Remove this line if there is no sample

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Open-and-save-Word-document-in-WinUI.md [hunk 8]
```diff
@@ -0,0 +1,267 @@
...
+{% endhighlight %}
+
+{% endtabs %}
+
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Read-and-Save-document/Open-and-save-Word-document/WinUI/WinUI-Desktop-app).
+
+By executing the program, you will get the **Word document** as follows.
+
+![WinUI Desktop open and save output Word document](WinUI_Images/OpenAndSaveOutput.png)
+
```
[the remark this answers is not in the document]
Comment (sharonaumao) [reply File-Formats/DocIO/Open-and-save-Word-document-in-WinUI.md:116]: @MohanaselvamJothi -So should the helper method also be published in UG?

**Review by Suriya-Balamurugan — APPROVED** [review]
I don't have any concerns in this code changes.

**Pull request merged**